cnxarchive/tests/views/test_content.py

```diff
@@ -1103,6 +1104,14 @@ def test_extra_w_utf8_characters(self):
                          'application/json')
         output['canPublish'].sort()
         self.assertEqual(output, {
+            u'books': [{
+                u'authors': [u'OpenStaxCollege'],
```

Comment:
Depending on where this is to be used, the author id is only minimally useful without the human readable information with it (e.g. fullname, title etc.).

As coded it's fine. We'll want to correct this in a followup changeset.